### DISCUSSION TITLE
Resize into smaller sd card

### DISCUSSION THREAD

**Discussion by mirko8054** [Drewsif/PiShrink#259] [Q&A] [answered]
Hello, i wanted to know if pishrink can be used to resize a pi image of 64gb sd card into 32gb, the used space barely goes to 20gb

**Comment by Drewsif** [comment, marked as answer]
More thank likely yes. It can not shrink it to exactly 32gb, but it will get it as small as possible.

**Reply by mirko8054** [reply]
Thanks for your answer, by this i know it is possible to downgrade the SD card size, however i am facing a problem, when i start the shrink of the image, before the process begins i get A LOT of errors that most likely goes like this "Found an error in sector XXXXX" Fixed? Yes" and so on, until it begins to shrink, it says *Warning Filesystem was modified*

**Reply by mirko8054** [reply]
[pishrink.log](https://github.com/Drewsif/PiShrink/files/10804611/pishrink.log)
i have uploaded you logs, can it be the fact that the starting image is corrupted?

**Reply by Drewsif** [reply]
From the log it looks like there was something wrong with the starting image. If you took the dd image while the Pi was online I highly recommend you do it offline from another machine.

It does look like it was successful though, from all the errors I'm not sure if it will work but you never know.
